guides/doc-Release_notes/redmine_release_notes

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+
+# Find the ""Release notes"" in said version
+# Pull all the release info from Redmine
+
+require 'uri'
+require 'net/http'
+require 'json'
+
+HOST = 'projects.theforeman.org'
+
+def build_uri(path, query = nil)
+  URI::HTTPS.build(host: HOST, path: path, query: query ? URI.encode_www_form(query) : nil)
+end
+
+def url_to_json(path, query = nil)
+  uri = build_uri(path, query)
+  response = Net::HTTP.get(uri)
+  JSON.parse(response)
+end
+
+# Determine version
+def get_version_id(project, release_name)
+  release_name = "Katello #{release_name}" if project == 'katello'
+
+  result = url_to_json("/projects/#{project}/versions.json")
+  return nil if result['versions'].nil?
+  result = result['versions'].detect{|v| v['name'] == release_name }
+  return nil if result.nil?
+  result['id']
+rescue StandardError => e
+  puts "Error getting version list from #{path}: #{e}"
+  exit 1
+end
+
+def gather_issues(release_id, offset = 0)
+  query = {'status_id' => 'closed', 'offset' => offset, 'limit' => 100, 'f[]' => 'cf_12', 'op[cf_12]' => '=', 'v[cf_12][]' => release_id}.to_a
+  result = url_to_json('/issues.json', query)
+  raise StandardError, result['errors'] if result['errors']
+
+  if result['total_count'].to_i - offset - 100 <= 0
+    result['issues']
+  else
+    result['issues'] += gather_issues(release_id, offset + 100)
+  end
+end
+
+def category_from_issue(issue)
+  category_name = issue.dig('category', 'name')
+
+  return unless category_name
+
+  case issue['project']['name']
+  when 'Smart Proxy'
+    return if category_name == 'Core'
+  when 'SELinux'
+    return if category == 'General Foreman'
+  end
+
+  category_name
+end
+
+def format_issue(issue)
+  subject = issue['subject'].gsub('`','\\\`').gsub('<','&lt;').gsub('>','&gt;').gsub('*','\\\*').gsub("'", "\\\'")
```

Comment:
I struggled with this. Right now the `'` escaping is wrong. However, I found that if you don't escape it, it ends interpreting. If you do, it ends up as `\'`.

Reply:
I was reading https://docs.antora.org/antora/2.3/asciidoc/quotes-and-apostrophes/ but I can't quite figure out how to literally display what was there. I ended up removing the escaping and manually fixing it up where needed.